tcadmin/appconfig.py

```diff
@@ -100,6 +100,13 @@ class AppConfig:
     # treated as relative to the current directory.
     check_path = attr.ib(type=str, init=False, default="checks")
 
+    # utilities for resource attr.ib's
+    description_prefix = attr.ib(
+        type=str,
+        init=False,
+        default="*DO NOT EDIT* - This resource is configured automatically.",
+    )
```

Comment:
This looks good!  Can you add something to the README so that users know this option exists?

Reply:
I would love to, but where exactly, will it go in README ?

Reply:
https://github.com/taskcluster/tc-admin#appconfig

Reply:
I am wondering what should I write in the description of the `description_prefix`.
Like it is customizable, then where you can customize it (the code), what more can I add about it?

Reply:
It would be customized in tc-admin.py, similar to `check_path`, so you can look to the docs for that parameter as an example.

Reply:
Yes sure, I will look into it.
Thank you